src/Methods.php

```diff
@@ -122,6 +123,33 @@ public static function method(array $methodData, Onfleet $api, ...$args)
 			$url = "{$url}{$httpQueryParams}";
 		}
 
+		// Reference https://docs.onfleet.com/reference/delivery-manifest
+		if ($deliveryManifestObject && isset($args[0]) && count($args) > 0) {
+			foreach ($args as $item) {
+				if (isset($item['hubId']) && isset($item['workerId'])) {
+					$body = array(
+						'path' => 'providers/manifest/generate?hubId=' . $item['hubId'] . '&workerId=' . $item['workerId'],
+						'method' => "GET"
+					);
+					$hasBody = true;
+				}
+				if (isset($item['googleApiKey'])) {
+					$api->api->headers["X-API-Key"] = 'Google ' . $item['googleApiKey'];
```

Comment:
This didn't work for me to get the `turnByTurn` and `totalDistance`

I checked how the headers where being sent and they looked like this:
```
Array
(
    [0] => Content-Type: application/json
    [1] => User-Agent: onfleet/php-onfleet-
    [2] => Authorization: Basic <onfleet-api-key>
    [X-API-Key] => Google <google-api-key>
)
```
It started working when I changed the line to this: 
`$api->api->headers[] = 'X-API-Key: Google ' . $item['googleApiKey'];`
That way, headers will look like this:
```
Array
(
    [0] => Content-Type: application/json
    [1] => User-Agent: onfleet/php-onfleet-
    [2] => Authorization: Basic  <onfleet-api-key>
    [3] => X-API-Key: Google <google-api-key>
)
```